### DISCUSSION TITLE
ISP Firewall blocking Jamulus public server list loading

### DISCUSSION THREAD

**Discussion by henderjm** [jamulussoftware/jamulus#1478] [Technical Support & Help]
Hi, 

I was trying to help a friend access Jamulus Public Servers through the home broadband. What we observed was the servers loaded when they used their mobile as a hotspot but the servers didn't load when we used the home broadband. What is probably obvious here is that mobile network is with a different provider to the home broadband.

The only way I could get the servers to load on the home broadbad, was to disable the firewall on the router but that is not exactly something I'm comfortable with at all as a long term solution.

I guess Port Mapping is the next best solution but I've no idea how to configure it in this sense because I'm a little confused about the whole architecture and how the servers are actually loading in the client.

Any advice would be great.

**Comment by gilgongo** [comment]
If the router has a "Block Fragmented IP Packets” option, then that should not be checked. If not, then it sounds like you are out of luck unless they can get another router. You can try port forwarding, but I doubt it would have any effect if the router isn't allowing the traffic.
